qa/rpc-tests/pruning.py

```diff
@@ -212,6 +227,90 @@ def reorg_back(self):
         # Verify we can now have the data for a block previously pruned
         assert(self.nodes[2].getblock(self.forkhash)["height"] == self.forkheight)
 
+    def manual_test(self):
+        self.nodes[3] = start_node(3, self.options.tmpdir, ["-debug=0","-prune=1"], timewait=900)
+        assert_equal(self.nodes[3].getblockcount(), 995)
+
+        # should not prune because chain tip of node 3 (995) < PruneAfterHeight (1000)
+        try:
+            self.nodes[3].pruneblockchain(500)
```

Comment:
Might be good to check for the specific error messages here and below, e.g. `assert_raises_message(JSONRPCException, "Blockchain is too short for pruning.", self.nodes[3].pruneblockchain, 500)`